modRlp/test/org/aion/rlp/UtilsTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ *     This file is part of the aion network project.
+ *
+ *     The aion network project is free software: you can redistribute it
+ *     and/or modify it under the terms of the GNU General Public License
+ *     as published by the Free Software Foundation, either version 3 of
+ *     the License, or any later version.
+ *
+ *     The aion network project is distributed in the hope that it will
+ *     be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ */
+package org.aion.rlp;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.Arrays;
+import org.aion.base.util.Hex;
+import org.junit.Test;
+
+public class UtilsTest {
```

Comment:
Could we also have tests for concatenating two empty arrays as well as empty with non-empty arrays?

Reply:
done in PR #572 commit ba5c2954639c95eee8258f5d9c4392d1ad147225